config/default/configmap/kfservice.yaml

```diff
@@ -39,3 +39,8 @@ data:
            "s3SecretAccessKeyName": "awsSecretAccessKey"
        }
     }
+  ingress: |-
+    {
+        "ingressGateway" : "knative-ingress-gateway.knative-serving",
+        "ingressService" : "istio-ingressgateway.istio-system.svc.cluster.local"
```

Comment:
Can we add the local gateway `cluster-local-gateway.knative-serving` as other knative provisioned virtual services do? The service may get called from in cluster.

Reply:
Created a separate issue so we can get this in and paralleize remaining items?

Reply:
this is the istio service that is configured against the lb.. e.g.
```
$ k get svc -n istio-system
NAME                   TYPE           CLUSTER-IP    EXTERNAL-IP      PORT(S)                                                                                                                                      AGE
istio-ingressgateway   LoadBalancer   10.0.219.27   104.210.45.165   15020:32245/TCP,80:31380/TCP,443:31390/TCP,31400:31400/TCP,15029:32334/TCP,15030:31737/TCP,15031:31719/TCP,15032:30893/TCP,15443:31661/TCP   19d
istio-pilot            ClusterIP      10.0.135.70   <none>           15010/TCP,15011/TCP,8080/TCP,15014/TCP                                                                                                       19d
```